client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -433,6 +433,39 @@ $.entwine('ss', function($){
     }
   });
 
+  // Hack: silverstripe/versioned coupling
+  $('.cms-edit-form .btn-toolbar button[name=action_doUnpublish].btn.action').entwine({
+    onclick: function(e) {
+      var owners = this.data('owners');
+      if (owners && parseInt(owners) > 0) {
+        var message = [
+          i18n.inject(
+            i18n._t(
+              'Admin.OWNED_WARNING_1',
+              'You are unpublishing content that is being used in {count} other published section(s).',
```

Comment:
Trailing `,` is out of place here I think.

Reply:
it's a valid trailing comma, in fact a future eslint rule ENFORCES trailing commas for multi-line params

Reply:
So weird.